Calendar/CalLogHelpers/TimelineFunctions.ps1

```diff
@@ -60,448 +60,48 @@ function FindFirstMeeting {
-                }
-                default {
-                    [array] $Output = "[$($CalLog.ResponsibleUser)] $($CalLog.TriggerAction) the Meeting with $($CalLog.Client)."
-                    $script:MeetingSummaryNeeded = $False
-                }
-            }
-        }
-        Cancellation {
-            switch ($CalLog.Client) {
-                Transport {
-                    if ($CalLog.IsException -eq $True) {
-                        [array] $Output = "Transport $($CalLog.TriggerAction)d a Meeting Cancellation based on changes by [$($CalLog.SenderSMTPAddress)] for the exception starting on [$($CalLog.StartTime)]"
-                    } else {
-                        [array] $Output = "Transport $($CalLog.TriggerAction)d a Meeting Cancellation based on changes by [$($CalLog.SenderSMTPAddress)]."
-                    }
-                }
-                ResourceBookingAssistant {
-                    if ($CalLog.TriggerAction -eq "MoveToDeletedItems") {
-                        [array] $Output = "ResourceBookingAssistant Deleted the Cancellation."
-                    } else {
-                        [array] $Output = "ResourceBookingAssistant $($CalLog.TriggerAction)d the Cancellation."
-                    }
-                }
-                default {
-                    if ($CalLog.IsException -eq $True) {
-                        [array] $Output = "[$($CalLog.ResponsibleUser)] $($CalLog.TriggerAction)d a Cancellation with $($CalLog.Client) for the exception starting on [$($CalLog.StartTime)]."
-                    } elseif ($CalLog.CalendarItemType -eq "RecurringMaster") {
-                        [array] $Output = "[$($CalLog.ResponsibleUser)] $($CalLog.TriggerAction)d a Cancellation for the Series with $($CalLog.Client)."
-                    } else {
-                        [array] $Output = "[$($CalLog.ResponsibleUser)] $($CalLog.TriggerAction)d the Cancellation with $($CalLog.Client)."
-                    }
-                }
-            }
-        }
-        default {
-            if ($CalLog.TriggerAction -eq "Create") {
-                $Action = "New "
-            } else {
-                $Action = "$($CalLog.TriggerAction)"
-            }
-            [array] $Output = "[$($CalLog.ResponsibleUser)] performed a $($Action) on the $($CalLog.ItemClass) with $($CalLog.Client)."
-        }
-    }
-
-    return $Output
-}
-
 function BuildTimeline {
-    param (
-        [string] $Identity
-    )
-    $ThisMeetingID = $script:GCDO.CleanGlobalObjectId | Select-Object -Unique
-    $ShortMeetingID = $ThisMeetingID.Substring($ThisMeetingID.length - 6)
-    if ($Identity -like "*@*") {
-        $ShortName = $Identity.Split('@')[0]
-    }
-    $ShortName = $ShortName.Substring(0, [System.Math]::Min(20, $ShortName.Length))
-    $Script:TimeLineFilename = "$($ShortName)_TimeLine_$ShortMeetingID.csv"
+    $script:TimeLineOutput = @()
```

Comment:
If this list ends up having a lot of entries in it, would recommend doing `System.Collections.Generic.List` as they are faster to add to and have less of a perf impact.